packages/ui/organisms/Frame/Frame.ts

```diff
@@ -218,7 +224,7 @@ export class Frame<T extends BaseProps = BaseProps> extends Base<T & FrameProps>
   /**
    * Fetch the given url.
    */
-  async fetch(url: string): Promise<string> {
+  async fetch(url: string, formData: FormData = null): Promise<string> {
     const cached = cache.get(url);
 
     if (cached) {
```

Comment:
yeah definetly, ran into some problems in my live code because of the caching when using multiple frames on the same page; this comment helpped me to identify the issue fast, thx :)